A72-open-telemetry-tracing.md

```diff
@@ -36,98 +38,125 @@ and OpenCensus's binary format for (de)serialization. The header name is unique
-number of preceding attempts for the RPC, and attribute `transparent-retry` that 
-shows whether stream is a transparent retry.
-* When the stream is created on transport, annotate delayed load balancer pick 
-complete, if any.
+* When span is created, add attribute `previous-rpc-attempts` that captures the
+  number of preceding attempts for the RPC, and attribute `transparent-retry` that
+  shows whether stream is a transparent retry.
+* When the stream is created on transport, annotate delayed load balancer pick
+  complete, if any.
 * When an outbound message has been sent, add message events to capture seq no.,
-type(SENT), uncompressed message size, and compressed message size if any 
-compression. The seq no. is a sequence of integer numbers starting from 0 
-to identify sent messages within the stream. The size is the total attempt message 
-bytes without encryption, not including grpc or transport framing bytes.
-* When an inbound message has been received from the transport, add message 
-events to capture seq no., type(Received), wire message size, and uncompressed
-message size if any decompression. The seq no. is a sequence of integer numbers 
-starting from 0 to identify received messages within the stream.
-* When the stream is closed, end the attempt span with RPC status.
+  type(SENT), uncompressed message size, and compressed message size if any
+  compression. The seq no. is a sequence of integer numbers starting from 0
+  to identify sent messages within the stream. The size is the total attempt message
+  bytes without encryption, not including grpc or transport framing bytes.
+* When an inbound message has been received from the transport, add message
+  events to capture seq no., type(Received), wire message size, and uncompressed
+  message size if any decompression. The seq no. is a sequence of integer numbers
+  starting from 0 to identify received messages within the stream.
+* When the stream is closed, set RPC status and end the attempt span.
 
 At the server:
 * When an outbound message has been sent, add message events to capture seq no.,
-type(SENT) and uncompressed message size, and compressed message size if any compression.
+  type(SENT) and uncompressed message size, and compressed message size if any compression.
 * When an inbound message has been read from the transport, add message events
-to capture seq no., type(Received), wire message size, and uncompressed message size,
-if any decompression.
-* When the stream is closed, end the span with RPC status.
+  to capture seq no., type(Received), wire message size, and uncompressed message size,
+  if any decompression.
+* When the stream is closed, set the RPC status and end the span.
 
-Note that C++ is missing the seq no. information. And Java has issue of reporting 
-decompressed message size upon receiving messages, 
+Note that C++ is missing the seq no. information. And Java has issue of reporting
```

Comment:
Is this paragraph talking about limitations in the existing OC impl or limitations that we're encoding into the new OT functionality?  If the former, it should probably just be removed, or maybe moved down to the "Migrate from OpenCensus to OpenTelemetry" section.  If the latter, we need to explain why we're going to have these limitations.